tests/dns/task-7018-dns-ids-stream-rule/test.yaml

```diff
@@ -0,0 +1,233 @@
+args:
+- -k none
+
+pcap: ../task-7018-ids-dns-keywords/input.pcap
+
+checks:
+- filter:
+    min-version: 7
+    count: 2
+    match:
+      event_type: alert
+- filter:
+    min-version: 8
+    count: 1
+    match:
+      event_type: alert
+      pcap_cnt: 7
+      alert.signature: DNS suricata query - payload rule
+      alert.signature_id: 1
+      proto: TCP
+      src_ip: 10.16.1.11
+      src_port: 36926
+      app_proto: dns
+      dest_ip: 9.9.9.9
+      dest_port: 53
+      direction: to_server
```

Comment:
Should we add transaction data here? Suricata commit bb45ac71ef572acb5591c135eb3c73e901a1cc51 changes which transaction gets logged here, which I think is an important deal of this test.

Reply:
Will do. Wasn't sure what would be the final decision on logging tx data for stream rules, so ended up not having that in the checks, but will bring them back.